bin/stencil-bundle

```diff
@@ -57,277 +42,7 @@ if (!themeConfig.configExists()) {
-        files = files.map(function (file) {
-            return file.replace(templatesBasePath + Path.sep, '').replace('.html', '');
-        });
-
-        Async.map(files, TemplateAssembler.assemble, function (err, results) {
-            var ret = {};
-
-            if (err) {
-                return callback(err);
-            }
-
-            _.each(files, function (file, index) {
-                ret[file] = results[index];
-            });
-
-            // Checks to see if there is a cyclical dependency
-            try {
-                new Cycles(results).detect();
-            } catch (e) {
-                console.error('error'.red);
-                throw e;
-            }
-
-            validator.validateObjects(results, function (err, validate) {
-                if (err) {
-                    console.error('error '.red + err.message);
-                    return callback(err);
-                }
-
-                console.log('ok'.green + ' -- Template Parsing Finished');
-
-                callback(null, ret);
-            });
-        });
-    });
-}
-
-function assembleSchema(callback) {
-    console.log('Building Theme Schema File...');
-
-    themeConfig.getSchema(function (err, schema) {
-        console.log('ok'.green + ' -- Theme Schema Building Finished');
-
-        callback(err, schema);
-    });
-}
-
-function assembleLangTask(callback) {
-    console.log('Language Files Parsing Started...');
-
-    if (!fileExist(LangAssembler.localeDirectory)) {
-        console.log('warning'.yellow + ' -- lang folder not found, translations will be disabled');
-        return callback(null, {});
-    }
-
-    LangAssembler.assemble(function (err, results) {
-        if (err) {
-            return callback(err);
-        }
-
-        console.log('ok'.green + ' -- Language Files Parsing Finished');
-        callback(null, results);
-    });
-}
-
-function getJspmBundleTask(jspmConfig) {
-    return function (callback) {
-        var oldConsoleError = console.error;
-
-        console.log('JavaScript Bundling Started...');
-
-        // Need to suppress annoying errors from Babel.
-        // They will be gone in the next minor version of JSPM (0.16.0).
-        // Until then, this will stay in place
-        console.error = function (error) {
-            if (!/Deprecated option metadataUsedHelpers/.test(error)) {
-                oldConsoleError(error);
-            }
-        };
-
-        Jspm.setPackagePath(themePath);
-        Jspm.bundleSFX(jspmConfig.bootstrap, jspmConfig.tmpBundleFile, {
-            minify: true,
-            mangle: true
-        }).then(function () {
-            console.log('ok'.green + ' -- JavaScript Bundling Finished');
-            console.error = oldConsoleError;
-            callback(null, true);
-        });
-    }
-}
+var bundle = new Bundle(themePath, themeConfig, configuration, stencilConfig, {cmd: Program});
```

Comment:
should declare variable up above and maybe put each argument on its own line because there are so many